modules/building/pages/prefetching-dependencies.adoc

```diff
@@ -1,8 +1,8 @@
 = Prefetching package manager dependencies for hermetic builds
 
-In {ProductName}, you can run a hermetic build by restricting network access to the build, but without network a build can’t fetch component dependencies from a repository and might fail. To avoid that, {ProductName} can prefetch dependencies for your hermetic builds using link:https://github.com/containerbuildsystem/cachi2/blob/main/README.md[Cachi2].
+In {ProductName}, you can run a hermetic build by restricting network access to the build; but without network access, a build will not be able to fetch component dependencies from repote repositories, and might fail. To avoid that, {ProductName} can prefetch dependencies for your hermetic builds using link:https://hermetoproject.github.io/hermeto[Hermeto].
```

Comment:
```suggestion
In {ProductName}, you can run a hermetic build by restricting network access to the build; but without network access, a build will not be able to fetch component dependencies from remote repositories, and might fail. To avoid that, {ProductName} can prefetch dependencies for your hermetic builds using link:https://hermetoproject.github.io/hermeto[Hermeto].
```

Reply:
Fixed